Source/WebKit/Shared/IPCTesterReceiverSwiftUtilities.h

```diff
@@ -25,30 +25,8 @@
 
 #pragma once
 
-#if ENABLE(WEBXR_HIT_TEST)
+#include <cstdint>
+#include <wtf/CompletionHandler.h>
+#include <wtf/RefCountable.h>
 
-#include "DOMPointInit.h"
-#include <wtf/Vector.h>
-
-namespace WebCore {
-
-struct FakeXRWorldInit {
-    enum class RegionType {
-        Point,
-        Plane,
-        Mesh
-    };
-    struct TriangleInit {
-        Vector<DOMPointInit> vertices;
-    };
-    struct RegionInit {
-        Vector<TriangleInit> faces;
-        RegionType type;
-    };
-
-    Vector<RegionInit> hitTestRegions;
-};
-
-} // namespace WebCore
-
-#endif // ENABLE(WEBXR_HIT_TEST)
+using IntCompletionHandler = WTF::RefCountable<WTF::CompletionHandler<void(uint32_t)>>;
```

Comment:
I think this file should be named like “IPCTesterReceiverTypes” or “IPCTesterReceiverBridging” since it’s a pet peeve of mine when a file name has a language name in it, and also if it has “Utilities” in it 😛

Reply:
I've renamed it to `IPCTesterReceiverSwiftTypes`. I agree 'utilities' is no good.
However, `IPCTesterReceiverSwift` is the name of the test message handler, as distinct from the pre-existing `IPCTesterReceiver` which receives and handles similar messages in C++. I could `s/Swift/Two/g` or something. I'll think about that.